### PR TITLE
project1

### PR DESCRIPTION
**Pull request by lakshmisindhu9100733271**
completed upto authentication

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            return render_template("register.html",message=error)
+    return render_template("register.html")
+
+@app.route("/login.html",methods=["GET","POST"])
+def login():
+    return render_template("login.html")
+
+@app.route("/admin")
+def admin():
+    users=User.query.all()
```
Comment (manasa133) [inline project1/application.py:45]: display the latest records first

File: project1/application.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            # print (passw,"====> entered password")
+
+            if passw == u.password :
+            #     print ("your mail id is found")
+                session["email"]=mail
+                return render_template("account.html")
+            else:
+                return "invalid password"
+        else:
+            return "No account associated with this password"
```
Comment (manasa133) [inline project1/application.py:67]: display these messages on the template

File: project1/application.py (hunk 1 above)
Comment (lakshmisindhu9100733271) [reply project1/application.py:45]: here I order it by desc order mam

File: project1/application.py (hunk 2 above)
Comment (lakshmisindhu9100733271) [reply project1/application.py:67]: here I created an other file name error.html and display them

Comment (manasa133) [reply project1/application.py:67]: dont create a template, just render the same template with this message

**Pull request merged**